errai-data-binding/src/main/java/org/jboss/errai/databinding/rebind/DataBindingUtil.java

```diff
@@ -413,43 +416,37 @@ public static boolean isBindableType(final MetaClass type) {
     return refreshedTypes;
   }
 
-  private static Set<MetaClass> findConfiguredBindableTypes() {
-    final Set<MetaClass> bindableTypes = new HashSet<>();
+  /**
+   * Reads nonbindable types from all ErraiApp.properties files on the classpath.
+   *
+   * @return a set of meta classes representing the configured nonbindable types.
+   */
+  public static Set<MetaClass> getConfiguredNonBindableTypes() {
+    if (configuredNonBindableTypes == null) {
```

Comment:
In `getConfiguredBindableTypes` there is a "refresh" case when `configuredBindableTypes` has already been loaded. This is important for making things work properly during refreshes of Super Dev Mode. In this case you *might* be okay avoiding this step since we are explicitly not using non-bindable types to generate anything, but this should be tested.